### ISSUE TITLE
It has been taking too long to run canu over 24 hours. Should I wait or change my command line

### ISSUE THREAD

**Issue by NguyenLePhuong** [marbl/canu#1460]
Dear experts, 
I am newbie in canu, I set up canu via bioconda. 
I am using ubuntu 16.04, 16Gb, core 5th generation intel i7 2.9 Hz, my remaining hard disk is 71 Gb before running canu command.

It has been over 24 hours and it seems that it won't run any more, it stuck at this step for 8 hours. 

"- Configured 6 overlapInCore jobs.
-- overlapInCore attempt 1 begins with 0 finished, and 6 to compute.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 03:32:24 2019 with 24.135 GB free disk space (6 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/overlap.sh 1 > /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/overlap.000001.out 2>&1"

Have you ever met this issue? I would like to ask what will be the solutions for me at this stage. Additionally, if I use Crtl + Z to exit the job, how can I return back without running from the begining?

Below is the full canu log on terminal. 

Thank you so much for your precious time and valuable help,

Yours sincerely,
Le Phuong.

canu  -p BC01  -d /home/rosh2/minionphage/BC01out2  genomeSize=0.12g  -nanopore-raw /home/rosh2/minionphage/BC01/*.fastq.gz  
-- Canu v0.0 (+0 commits) r0 unknown-hash-tag-no-repository-available.
-- Detected Java(TM) Runtime Environment '1.8.0_92' (from '/home/rosh2/miniconda3/bin/java').
-- Detected gnuplot version '5.0 patchlevel 3' (from 'gnuplot') and image format 'png'.
-- Detected 8 CPUs and 16 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'bogart (unitigger)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    6 GB memory for stage 'read error detection (overlap error adjustment)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap error adjustment'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'utgcns (consensus)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap store parallel bucketizer'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    8 GB memory for stage 'overlap store parallel sorting'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'meryl (k-mer counting)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    8 GB memory for stage 'falcon_sense (read correction)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
--
-- This is canu parallel iteration #1, out of a maximum of 2 attempts.
--
-- Final error rates before starting pipeline:
--   
--   genomeSize          -- 120000000
--   errorRate           -- 0.048
--   
--   corOvlErrorRate     -- 0.144
--   obtOvlErrorRate     -- 0.144
--   utgOvlErrorRate     -- 0.144
--   
--   obtErrorRate        -- 0.144
--   
--   cnsErrorRate        -- 0.144
--
--
-- BEGIN CORRECTION
--
----------------------------------------
-- Starting command on Tue Aug 27 22:27:44 2019 with 49.94 GB free disk space

    /home/rosh2/miniconda3/libexec/gatekeeperCreate \
      -minlength 1000 \
      -o /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore.BUILDING \
      /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore.gkp \
    > /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore.BUILDING.err 2>&1

-- Finished on Tue Aug 27 22:27:59 2019 (15 seconds) with 49.735 GB free disk space
----------------------------------------
--
-- In gatekeeper store '/home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore':
--   Found 62580 reads.
--   Found 701008082 bases (5.84 times coverage).
--
--   Read length histogram (one '*' equals 169.9 reads):
--        0    999      0 
--     1000   1999  11893 **********************************************************************
--     2000   2999   7396 *******************************************
--     3000   3999   5240 ******************************
--     4000   4999   4281 *************************
--     5000   5999   3486 ********************
--     6000   6999   2908 *****************
--     7000   7999   2449 **************
--     8000   8999   2107 ************
--     9000   9999   1737 **********
--    10000  10999   1549 *********
--    11000  11999   1435 ********
--    12000  12999   1196 *******
--    13000  13999   1063 ******
--    14000  14999    957 *****
--    15000  15999    900 *****
--    16000  16999    800 ****
--    17000  17999    774 ****
--    18000  18999    689 ****
--    19000  19999    680 ****
--    20000  20999    624 ***
--    21000  21999    631 ***
--    22000  22999    565 ***
--    23000  23999    565 ***
--    24000  24999    532 ***
--    25000  25999    525 ***
--    26000  26999    425 **
--    27000  27999    428 **
--    28000  28999    445 **
--    29000  29999    406 **
--    30000  30999    356 **
--    31000  31999    303 *
--    32000  32999    294 *
--    33000  33999    288 *
--    34000  34999    255 *
--    35000  35999    237 *
--    36000  36999    214 *
--    37000  37999    182 *
--    38000  38999    151 
--    39000  39999    160 
--    40000  40999    150 
--    41000  41999    133 
--    42000  42999    112 
--    43000  43999    114 
--    44000  44999     87 
--    45000  45999     90 
--    46000  46999     82 
--    47000  47999     96 
--    48000  48999     69 
--    49000  49999     66 
--    50000  50999     56 
--    51000  51999     66 
--    52000  52999     46 
--    53000  53999     53 
--    54000  54999     62 
--    55000  55999     93 
--    56000  56999    274 *
--    57000  57999   1199 *******
--    58000  58999    511 ***
--    59000  59999     20 
--    60000  60999     21 
--    61000  61999     10 
--    62000  62999      9 
--    63000  63999      4 
--    64000  64999      2 
--    65000  65999      3 
--    66000  66999      2 
--    67000  67999      2 
--    68000  68999      3 
--    69000  69999      0 
--    70000  70999      2 
--    71000  71999      3 
--    72000  72999      2 
--    73000  73999      0 
--    74000  74999      1 
--    75000  75999      0 
--    76000  76999      0 
--    77000  77999      1 
--    78000  78999      1 
--    79000  79999      0 
--    80000  80999      1 
--    81000  81999      0 
--    82000  82999      0 
--    83000  83999      0 
--    84000  84999      0 
--    85000  85999      0 
--    86000  86999      0 
--    87000  87999      0 
--    88000  88999      0 
--    89000  89999      0 
--    90000  90999      0 
--    91000  91999      0 
--    92000  92999      0 
--    93000  93999      0 
--    94000  94999      0 
--    95000  95999      0 
--    96000  96999      0 
--    97000  97999      1 
--    98000  98999      0 
--    99000  99999      0 
--   100000 100999      1 
--   101000 101999      0 
--   102000 102999      0 
--   103000 103999      1 
--   104000 104999      0 
--   105000 105999      1 
--   106000 106999      0 
--   107000 107999      1 
--   108000 108999      0 
--   109000 109999      1 
--   110000 110999      0 
--   111000 111999      1 
--   112000 112999      0 
--   113000 113999      0 
--   114000 114999      0 
--   115000 115999      0 
--   116000 116999      1 
-- Meryl attempt 1 begins.
----------------------------------------
-- Starting concurrent execution on Tue Aug 27 22:27:59 2019 with 49.733 GB free disk space (1 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/correction/0-mercounts/meryl.sh 1 > /home/rosh2/minionphage/BC01out2/correction/0-mercounts/meryl.000001.out 2>&1

-- Finished on Tue Aug 27 22:28:53 2019 (54 seconds) with 49.318 GB free disk space
----------------------------------------
-- Meryl finished successfully.
----------------------------------------
-- Starting command on Tue Aug 27 22:28:53 2019 with 49.318 GB free disk space

    /home/rosh2/miniconda3/libexec/meryl \
      -Dh \
      -s /home/rosh2/minionphage/BC01out2/correction/0-mercounts/BC01.ms16 \
    > /home/rosh2/minionphage/BC01out2/correction/0-mercounts/BC01.ms16.histogram \
    2> /home/rosh2/minionphage/BC01out2/correction/0-mercounts/BC01.ms16.histogram.info

-- Finished on Tue Aug 27 22:28:53 2019 (lickety-split) with 49.318 GB free disk space
----------------------------------------
-- For mhap overlapping, set repeat k-mer threshold to 7000.
--
-- Found 700069382 16-mers; 103668737 distinct and 77004393 unique.  Largest count 16335.
--
-- OVERLAPPER (mhap) (correction)
--
-- Set corMhapSensitivity=high based on read coverage of 5.
--
-- PARAMETERS: hashes=768, minMatches=2, threshold=0.73
--
-- Given 13 GB, can fit 19500 reads per block.
-- For 5 blocks, set stride to 2 blocks.
-- Logging partitioning to '/home/rosh2/minionphage/BC01out2/correction/1-overlapper/partitioning.log'.
-- Configured 4 mhap precompute jobs.
-- Configured 5 mhap overlap jobs.
-- mhap precompute attempt 1 begins with 0 finished, and 4 to compute.
----------------------------------------
-- Starting concurrent execution on Tue Aug 27 22:28:55 2019 with 49.73 GB free disk space (4 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.sh 1 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.000001.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.sh 2 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.000002.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.sh 3 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.000003.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.sh 4 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/precompute.000004.out 2>&1

-- Finished on Tue Aug 27 23:22:48 2019 (3233 seconds) with 47.947 GB free disk space
----------------------------------------
-- All 4 mhap precompute jobs finished successfully.
-- mhap attempt 1 begins with 0 finished, and 5 to compute.
----------------------------------------
-- Starting concurrent execution on Tue Aug 27 23:22:48 2019 with 47.947 GB free disk space (5 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.sh 1 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.000001.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.sh 2 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.000002.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.sh 3 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.000003.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.sh 4 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.000004.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.sh 5 > /home/rosh2/minionphage/BC01out2/correction/1-overlapper/mhap.000005.out 2>&1

-- Finished on Wed Aug 28 02:03:11 2019 (9623 seconds) with 36.418 GB free disk space
----------------------------------------
-- Found 5 mhap overlap output files.
----------------------------------------
-- Starting command on Wed Aug 28 02:03:11 2019 with 36.418 GB free disk space

    /home/rosh2/miniconda3/libexec/ovStoreBuild \
     -O /home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore.BUILDING \
     -G /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore \
     -M 2-8 \
     -L /home/rosh2/minionphage/BC01out2/correction/1-overlapper/ovljob.files \
     > /home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore.err 2>&1

-- Finished on Wed Aug 28 02:22:11 2019 (1140 seconds) with 11.19 GB free disk space
----------------------------------------
--
-- Overlap store '/home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore' successfully constructed.
--
-- Purged 15.269 GB in 20 overlap output files and 9 directories.
-- Overlap store '/home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore' statistics not available (skipped in correction and trimming stages).
----------------------------------------
-- Starting command on Wed Aug 28 02:22:12 2019 with 24.465 GB free disk space

    /home/rosh2/miniconda3/libexec/filterCorrectionOverlaps \
      -G /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore \
      -O /home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore \
      -S /home/rosh2/minionphage/BC01out2/correction/2-correction/BC01.globalScores.WORKING \
      -c 40 \
      -l 0 \
    > /home/rosh2/minionphage/BC01out2/correction/2-correction/BC01.globalScores.err 2>&1

-- Finished on Wed Aug 28 02:26:42 2019 (270 seconds) with 24.459 GB free disk space
----------------------------------------
----------------------------------------
-- Starting command on Wed Aug 28 02:26:42 2019 with 24.459 GB free disk space

    /home/rosh2/miniconda3/libexec/generateCorrectionLayouts \
      -G /home/rosh2/minionphage/BC01out2/correction/BC01.gkpStore \
      -O /home/rosh2/minionphage/BC01out2/correction/BC01.ovlStore \
      -S /home/rosh2/minionphage/BC01out2/correction/2-correction/BC01.globalScores \
      -C 80 \
      -p /home/rosh2/minionphage/BC01out2/correction/2-correction/BC01.estimate.WORKING

-- Finished on Wed Aug 28 02:30:27 2019 (225 seconds) with 24.457 GB free disk space
----------------------------------------
-- Set corMinCoverage=0 based on read coverage of 5.
-- Using overlaps no worse than 0.50 fraction error for correcting reads (from corErrorRate parameter).
-- generate corrected reads attempt 1 begins with 0 finished, and 7 to compute.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 02:30:30 2019 with 24.452 GB free disk space (7 processes; 2 concurrently)

    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 1 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000001.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 2 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000002.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 3 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000003.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 4 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000004.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 5 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000005.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 6 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000006.out 2>&1
    /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.sh 7 > /home/rosh2/minionphage/BC01out2/correction/2-correction/correctReads.000007.out 2>&1

-- Finished on Wed Aug 28 03:31:21 2019 (3651 seconds) with 23.934 GB free disk space
----------------------------------------
-- Found 7 read correction output files.
-- Found 7 read correction output files.
-- Concatenating correctReads output.
-- Analyzing correctReads output.
--
-- Purging correctReads output after merging to final output file.
-- Purged 7 .dump.success sentinels.
-- Purged 7 .fasta outputs.
-- Purged 7 .err outputs.
-- Purged 7 .out job log outputs.
--
-- Corrected reads saved in '/home/rosh2/minionphage/BC01out2/BC01.correctedReads.fasta.gz'.
--
--
-- BEGIN TRIMMING
--
----------------------------------------
-- Starting command on Wed Aug 28 03:31:33 2019 with 24.278 GB free disk space

    /home/rosh2/miniconda3/libexec/gatekeeperCreate \
      -minlength 1000 \
      -o /home/rosh2/minionphage/BC01out2/trimming/BC01.gkpStore.BUILDING \
      /home/rosh2/minionphage/BC01out2/trimming/BC01.gkpStore.gkp \
    > /home/rosh2/minionphage/BC01out2/trimming/BC01.gkpStore.BUILDING.err 2>&1

-- Finished on Wed Aug 28 03:31:40 2019 (7 seconds) with 24.137 GB free disk space
----------------------------------------
--
-- In gatekeeper store '/home/rosh2/minionphage/BC01out2/trimming/BC01.gkpStore':
--   Found 27596 reads.
--   Found 544941919 bases (4.54 times coverage).
--
--   Read length histogram (one '*' equals 18.12 reads):
--        0    999      0 
--     1000   1999    454 *************************
--     2000   2999   1011 *******************************************************
--     3000   3999   1161 ****************************************************************
--     4000   4999   1268 *********************************************************************
--     5000   5999   1269 **********************************************************************
--     6000   6999   1263 *********************************************************************
--     7000   7999   1178 ****************************************************************
--     8000   8999   1122 *************************************************************
--     9000   9999    991 ******************************************************
--    10000  10999    960 ****************************************************
--    11000  11999    934 ***************************************************
--    12000  12999    812 ********************************************
--    13000  13999    795 *******************************************
--    14000  14999    747 *****************************************
--    15000  15999    711 ***************************************
--    16000  16999    655 ************************************
--    17000  17999    644 ***********************************
--    18000  18999    566 *******************************
--    19000  19999    593 ********************************
--    20000  20999    554 ******************************
--    21000  21999    547 ******************************
--    22000  22999    511 ****************************
--    23000  23999    492 ***************************
--    24000  24999    466 *************************
--    25000  25999    480 **************************
--    26000  26999    383 *********************
--    27000  27999    389 *********************
--    28000  28999    420 ***********************
--    29000  29999    387 *********************
--    30000  30999    331 ******************
--    31000  31999    283 ***************
--    32000  32999    287 ***************
--    33000  33999    281 ***************
--    34000  34999    261 **************
--    35000  35999    221 ************
--    36000  36999    220 ************
--    37000  37999    184 **********
--    38000  38999    135 *******
--    39000  39999    168 *********
--    40000  40999    145 *******
--    41000  41999    137 *******
--    42000  42999    116 ******
--    43000  43999    103 *****
--    44000  44999     91 *****
--    45000  45999     88 ****
--    46000  46999     81 ****
--    47000  47999     85 ****
--    48000  48999     86 ****
--    49000  49999     70 ***
--    50000  50999     57 ***
--    51000  51999     62 ***
--    52000  52999     52 **
--    53000  53999     45 **
--    54000  54999     59 ***
--    55000  55999     86 ****
--    56000  56999    268 **************
--    57000  57999    874 ************************************************
--    58000  58999    855 ***********************************************
--    59000  59999     20 *
--    60000  60999     14 
--    61000  61999     16 
--    62000  62999     10 
--    63000  63999      7 
--    64000  64999      4 
--    65000  65999      3 
--    66000  66999      3 
--    67000  67999      1 
--    68000  68999      2 
--    69000  69999      3 
--    70000  70999      1 
--    71000  71999      1 
--    72000  72999      4 
--    73000  73999      1 
--    74000  74999      0 
--    75000  75999      1 
--    76000  76999      0 
--    77000  77999      1 
--    78000  78999      0 
--    79000  79999      1 
--    80000  80999      1 
--    81000  81999      0 
--    82000  82999      0 
--    83000  83999      0 
--    84000  84999      0 
--    85000  85999      0 
--    86000  86999      0 
--    87000  87999      0 
--    88000  88999      0 
--    89000  89999      0 
--    90000  90999      0 
--    91000  91999      0 
--    92000  92999      0 
--    93000  93999      0 
--    94000  94999      0 
--    95000  95999      0 
--    96000  96999      0 
--    97000  97999      0 
--    98000  98999      1 
--    99000  99999      0 
--   100000 100999      0 
--   101000 101999      1 
--   102000 102999      0 
--   103000 103999      0 
--   104000 104999      1 
--   105000 105999      0 
--   106000 106999      1 
--   107000 107999      0 
--   108000 108999      1 
--   109000 109999      0 
--   110000 110999      0 
--   111000 111999      0 
--   112000 112999      1 
--   113000 113999      1 
--   114000 114999      0 
--   115000 115999      0 
--   116000 116999      0 
--   117000 117999      1 
-- Meryl attempt 1 begins.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 03:31:41 2019 with 24.137 GB free disk space (1 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/meryl.sh 1 > /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/meryl.000001.out 2>&1

-- Finished on Wed Aug 28 03:32:22 2019 (41 seconds) with 23.662 GB free disk space
----------------------------------------
-- Meryl finished successfully.
----------------------------------------
-- Starting command on Wed Aug 28 03:32:22 2019 with 23.662 GB free disk space

    /home/rosh2/miniconda3/libexec/meryl \
      -Dh \
      -s /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22 \
    > /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.histogram \
    2> /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.histogram.info

-- Finished on Wed Aug 28 03:32:22 2019 (lickety-split) with 23.662 GB free disk space
----------------------------------------
----------------------------------------
-- Starting command on Wed Aug 28 03:32:22 2019 with 23.662 GB free disk space

    /home/rosh2/miniconda3/libexec/estimate-mer-threshold  \
      -m /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22  \
    > /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.estMerThresh.out \
    2> /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.estMerThresh.err

-- Finished on Wed Aug 28 03:32:22 2019 (lickety-split) with 23.662 GB free disk space
----------------------------------------
----------------------------------------
-- Starting command on Wed Aug 28 03:32:22 2019 with 23.662 GB free disk space

    /home/rosh2/miniconda3/libexec/meryl \
      -Dt \
      -n 5551 \
      -s /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22 \
    > /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.frequentMers.fasta \
    2> /home/rosh2/minionphage/BC01out2/trimming/0-mercounts/BC01.ms22.frequentMers.fasta.err

-- Finished on Wed Aug 28 03:32:24 2019 (2 seconds) with 23.661 GB free disk space
----------------------------------------
-- Reset obtOvlMerThreshold from auto to 5551.
--
-- Found 544362403 22-mers; 88933729 distinct and 74144272 unique.  Largest count 13477.
--
-- OVERLAPPER (normal) (trimming) erate=0.144
--
----------------------------------------
-- Starting command on Wed Aug 28 03:32:24 2019 with 24.136 GB free disk space

    /home/rosh2/miniconda3/libexec/overlapInCorePartition \
     -g  /home/rosh2/minionphage/BC01out2/trimming/BC01.gkpStore \
     -bl 100000000 \
     -bs 0 \
     -rs 2000000 \
     -rl 0 \
     -ol 500 \
     -o  /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/BC01.partition \
    > /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/BC01.partition.err 2>&1

-- Finished on Wed Aug 28 03:32:24 2019 (lickety-split) with 24.135 GB free disk space
----------------------------------------
--
-- Configured 6 overlapInCore jobs.
-- overlapInCore attempt 1 begins with 0 finished, and 6 to compute.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 03:32:24 2019 with 24.135 GB free disk space (6 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/overlap.sh 1 > /home/rosh2/minionphage/BC01out2/trimming/1-overlapper/overlap.000001.out 2>&1

**Comment by NguyenLePhuong** [comment]
Dear Skoren,

Thank you so much for your suggestion. I needs to be patience, since it took me 2 days to run this. I was wondered whether the computer is still running this job or it stopped because of some reasons. Therefore, I used Ctrl + Z to exit. it was kind of impatience. I would like to ask whether I can return to the previous stage.

Thank you very much,
Yours sincerely,
Le Phuong.

**Comment by NguyenLePhuong** [comment]
Dear Skoren,

I tried to return the job by copy and past the command line again. However, it did not go ahead with the previous run, instead, the process stopped at the initial process for a long time, I meant it did not show any process. I would like to ask whether there are any other ways to know the canu is still analyzing the data or it has been postponed without notification,

This is a newbie question, thank you for your patience in answer this,

Yours sincerely,
Le Phuong.

**Comment by NguyenLePhuong** [comment]
Dear Skoren,

Thank you very much, I was wrong, the canu continued the last step, it did not show the previous steps. 

Firstly, I run the command line;

rosh2@Roshan:~$ canu  -p BC01  -d /home/rosh2/minionphage/BC01out  genomeSize=0.12g  -nanopore-raw /home/rosh2/minionphage/BC01/*.fastq.gz  
-- Canu v0.0 (+0 commits) r0 unknown-hash-tag-no-repository-available.
-- Detected Java(TM) Runtime Environment '1.8.0_92' (from '/home/rosh2/miniconda3/bin/java').
-- Detected gnuplot version '5.0 patchlevel 3' (from 'gnuplot') and image format 'png'.
-- Detected 8 CPUs and 16 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'bogart (unitigger)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    6 GB memory for stage 'read error detection (overlap error adjustment)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap error adjustment'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'utgcns (consensus)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap store parallel bucketizer'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    8 GB memory for stage 'overlap store parallel sorting'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'meryl (k-mer counting)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    8 GB memory for stage 'falcon_sense (read correction)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
--
-- This is canu parallel iteration #1, out of a maximum of 2 attempts.
--
-- Final error rates before starting pipeline:
--   
--   genomeSize          -- 120000000
--   errorRate           -- 0.048
--   
--   corOvlErrorRate     -- 0.144
--   obtOvlErrorRate     -- 0.144
--   utgOvlErrorRate     -- 0.144
--   
--   obtErrorRate        -- 0.144
--   
--   cnsErrorRate        -- 0.144
--
--
-- BEGIN CORRECTION
--
----------------------------------------
-- Starting command on Wed Aug 28 21:33:49 2019 with 307.378 GB free disk space

    /home/rosh2/miniconda3/libexec/gatekeeperCreate \
      -minlength 1000 \
      -o /home/rosh2/minionphage/BC01out/correction/BC01.gkpStore.BUILDING \
      /home/rosh2/minionphage/BC01out/correction/BC01.gkpStore.gkp \
    > /home/rosh2/minionphage/BC01out/correction/BC01.gkpStore.BUILDING.err 2>&1

-- Finished on Wed Aug 28 21:34:04 2019 (15 seconds) with 307.173 GB free disk space
----------------------------------------
--
-- In gatekeeper store '/home/rosh2/minionphage/BC01out/correction/BC01.gkpStore':
--   Found 62580 reads.
--   Found 701008082 bases (5.84 times coverage).
--
--   Read length histogram (one '*' equals 169.9 reads):
--        0    999      0 
--     1000   1999  11893 **********************************************************************
--     2000   2999   7396 *******************************************
--     3000   3999   5240 ******************************
--     4000   4999   4281 *************************
--     5000   5999   3486 ********************
--     6000   6999   2908 *****************
--     7000   7999   2449 **************
--     8000   8999   2107 ************
--     9000   9999   1737 **********
--    10000  10999   1549 *********
--    11000  11999   1435 ********
--    12000  12999   1196 *******
--    13000  13999   1063 ******
--    14000  14999    957 *****
--    15000  15999    900 *****
--    16000  16999    800 ****
--    17000  17999    774 ****
--    18000  18999    689 ****
--    19000  19999    680 ****
--    20000  20999    624 ***
--    21000  21999    631 ***
--    22000  22999    565 ***
--    23000  23999    565 ***
--    24000  24999    532 ***
--    25000  25999    525 ***
--    26000  26999    425 **
--    27000  27999    428 **
--    28000  28999    445 **
--    29000  29999    406 **
--    30000  30999    356 **
--    31000  31999    303 *
--    32000  32999    294 *
--    33000  33999    288 *
--    34000  34999    255 *
--    35000  35999    237 *
--    36000  36999    214 *
--    37000  37999    182 *
--    38000  38999    151 
--    39000  39999    160 
--    40000  40999    150 
--    41000  41999    133 
--    42000  42999    112 
--    43000  43999    114 
--    44000  44999     87 
--    45000  45999     90 
--    46000  46999     82 
--    47000  47999     96 
--    48000  48999     69 
--    49000  49999     66 
--    50000  50999     56 
--    51000  51999     66 
--    52000  52999     46 
--    53000  53999     53 
--    54000  54999     62 
--    55000  55999     93 
--    56000  56999    274 *
--    57000  57999   1199 *******
--    58000  58999    511 ***
--    59000  59999     20 
--    60000  60999     21 
--    61000  61999     10 
--    62000  62999      9 
--    63000  63999      4 
--    64000  64999      2 
--    65000  65999      3 
--    66000  66999      2 
--    67000  67999      2 
--    68000  68999      3 
--    69000  69999      0 
--    70000  70999      2 
--    71000  71999      3 
--    72000  72999      2 
--    73000  73999      0 
--    74000  74999      1 
--    75000  75999      0 
--    76000  76999      0 
--    77000  77999      1 
--    78000  78999      1 
--    79000  79999      0 
--    80000  80999      1 
--    81000  81999      0 
--    82000  82999      0 
--    83000  83999      0 
--    84000  84999      0 
--    85000  85999      0 
--    86000  86999      0 
--    87000  87999      0 
--    88000  88999      0 
--    89000  89999      0 
--    90000  90999      0 
--    91000  91999      0 
--    92000  92999      0 
--    93000  93999      0 
--    94000  94999      0 
--    95000  95999      0 
--    96000  96999      0 
--    97000  97999      1 
--    98000  98999      0 
--    99000  99999      0 
--   100000 100999      1 
--   101000 101999      0 
--   102000 102999      0 
--   103000 103999      1 
--   104000 104999      0 
--   105000 105999      1 
--   106000 106999      0 
--   107000 107999      1 
--   108000 108999      0 
--   109000 109999      1 
--   110000 110999      0 
--   111000 111999      1 
--   112000 112999      0 
--   113000 113999      0 
--   114000 114999      0 
--   115000 115999      0 
--   116000 116999      1 
-- Meryl attempt 1 begins.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 21:34:05 2019 with 307.171 GB free disk space (1 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out/correction/0-mercounts/meryl.sh 1 > /home/rosh2/minionphage/BC01out/correction/0-mercounts/meryl.000001.out 2>&1

-- Finished on Wed Aug 28 21:34:57 2019 (52 seconds) with 306.758 GB free disk space
----------------------------------------
-- Meryl finished successfully.
----------------------------------------
-- Starting command on Wed Aug 28 21:34:57 2019 with 306.758 GB free disk space

    /home/rosh2/miniconda3/libexec/meryl \
      -Dh \
      -s /home/rosh2/minionphage/BC01out/correction/0-mercounts/BC01.ms16 \
    > /home/rosh2/minionphage/BC01out/correction/0-mercounts/BC01.ms16.histogram \
    2> /home/rosh2/minionphage/BC01out/correction/0-mercounts/BC01.ms16.histogram.info

-- Finished on Wed Aug 28 21:34:57 2019 (lickety-split) with 306.758 GB free disk space
----------------------------------------
-- For mhap overlapping, set repeat k-mer threshold to 7000.
--
-- Found 700069382 16-mers; 103668737 distinct and 77004393 unique.  Largest count 16335.
--
-- OVERLAPPER (mhap) (correction)
--
-- Set corMhapSensitivity=high based on read coverage of 5.
--
-- PARAMETERS: hashes=768, minMatches=2, threshold=0.73
--
-- Given 13 GB, can fit 19500 reads per block.
-- For 5 blocks, set stride to 2 blocks.
-- Logging partitioning to '/home/rosh2/minionphage/BC01out/correction/1-overlapper/partitioning.log'.
-- Configured 4 mhap precompute jobs.
-- Configured 5 mhap overlap jobs.
-- mhap precompute attempt 1 begins with 0 finished, and 4 to compute.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 21:34:59 2019 with 307.17 GB free disk space (4 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.sh 1 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.000001.out 2>&1
    /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.sh 2 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.000002.out 2>&1
    /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.sh 3 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.000003.out 2>&1
    /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.sh 4 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/precompute.000004.out 2>&1

-- Finished on Wed Aug 28 22:28:52 2019 (3233 seconds) with 305.409 GB free disk space
----------------------------------------
-- All 4 mhap precompute jobs finished successfully.
-- mhap attempt 1 begins with 0 finished, and 5 to compute.
----------------------------------------
-- Starting concurrent execution on Wed Aug 28 22:28:52 2019 with 305.409 GB free disk space (5 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.sh 1 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.000001.out 2>&1
    /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.sh 2 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.000002.out 2>&1
    /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.sh 3 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.000003.out 2>&1
^Z
[6]+  Stopped                 canu -p BC01 -d /home/rosh2/minionphage/BC01out genomeSize=0.12g -nanopore-raw /home/rosh2/minionphage/BC01/*.fastq.g


After that, I used Ctrl + Z to exit
And then, rerun again,
canu  -p BC01  -d /home/rosh2/minionphage/BC01out  genomeSize=0.12g  -nanopore-raw /home/rosh2/minionphage/BC01/*.fastq.gz  
-- Canu v0.0 (+0 commits) r0 unknown-hash-tag-no-repository-available.
-- Detected Java(TM) Runtime Environment '1.8.0_92' (from '/home/rosh2/miniconda3/bin/java').
-- Detected gnuplot version '5.0 patchlevel 3' (from 'gnuplot') and image format 'png'.
-- Detected 8 CPUs and 16 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'bogart (unitigger)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'mhap (overlapper)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    6 GB memory for stage 'read error detection (overlap error adjustment)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap error adjustment'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'utgcns (consensus)'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlap store parallel bucketizer'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    8 GB memory for stage 'overlap store parallel sorting'.
-- Allowed to run   8 jobs concurrently, and use up to   1 compute thread  and    2 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and    8 GB memory for stage 'overlapper'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   16 GB memory for stage 'meryl (k-mer counting)'.
-- Allowed to run   2 jobs concurrently, and use up to   4 compute threads and    8 GB memory for stage 'falcon_sense (read correction)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
-- Allowed to run   1 job  concurrently, and use up to   8 compute threads and   13 GB memory for stage 'minimap (overlapper)'.
--
-- This is canu parallel iteration #1, out of a maximum of 2 attempts.
--
-- Final error rates before starting pipeline:
--   
--   genomeSize          -- 120000000
--   errorRate           -- 0.048
--   
--   corOvlErrorRate     -- 0.144
--   obtOvlErrorRate     -- 0.144
--   utgOvlErrorRate     -- 0.144
--   
--   obtErrorRate        -- 0.144
--   
--   cnsErrorRate        -- 0.144
--
--
-- BEGIN CORRECTION
--
-- mhap attempt 1 begins with 2 finished, and 3 to compute.
----------------------------------------
-- Starting concurrent execution on Thu Aug 29 00:43:25 2019 with 294.97 GB free disk space (3 processes; 1 concurrently)

    /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.sh 3 > /home/rosh2/minionphage/BC01out/correction/1-overlapper/mhap.000003.out 2>&1


And it stopped there for long time, may be because my computer is not strong enough to run this,

Thank you so much for your precious time and instructions,

Yours sincerely,
Student Le Phuong.
